File: src/api/Livraria/Livraria.Domain/Entities/EmprestimoLivro.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Livraria.Domain.Entities
{
    public class EmprestimoLivro :BaseEntity 
    {
        public int UsuarioId { get; set; }
        public virtual Usuario Usuario { get; set; }
        public int LivroId { get; set; }
        public virtual Livro Livro { get; set; }
        public DateTime? DataDevolucao { get; set; }
        public bool Devolvido { get; set; }
    }
}

File: src/client/livraria-app/src/app/model/emprestimo-livro.ts
import { Usuario } from './usuario';
import { Livro } from './livro';

export interface EmprestimoLivro {
    usuarioId: number;
    usuario: Usuario;
    livroId: number;
    livro: Livro;
    dataDevolucao: Date;
    devolvido: boolean;
    id: number;
}